packages/dev-utils/package.json

```diff
@@ -6,9 +6,9 @@
     "license": "MIT",
     "bin": "./index.js",
     "devDependencies": {
-        "@svelte-add/ast-manipulation": "workspace:^",
-        "@svelte-add/ast-tooling": "workspace:^",
-        "@svelte-add/core": "workspace:^"
+        "@svelte-add/ast-manipulation": "workspace:*",
```

Comment:
I wonder if these should be left as `^` too since they're not being distributed to users

Reply:
Same thing as the website, it doesn't really matter which one it uses since it's not published, so it'll always use whatever versions in the current workspace